source/presentation/4.0/model.md

```diff
@@ -0,0 +1,2287 @@
+
+{% include api/code_header.html %}
+``` json-doc
+{ "summary": { "en": [ "<p>Short <b>summary</b> of the resource.</p>" ] } }
+```
+
+### JSON Description Availability
+
+JSON descriptions _SHOULD_ be embedded within the JSON of parent resources, and _MAY_ also be available via separate requests from the HTTP(S) URI given in the resource's `id` property. Links to Content Resources _MUST_ be given as a JSON object with the `id` and `type` properties and _SHOULD_ have `format` or `profile` to give a hint as to what sort of resource is being referred to.
+
+{% include api/code_header.html %}
+``` json-doc
+{
+  "rendering": [
+    {
+      "id": "https://example.org/content/book.pdf",
+      "type": "Text",
+      "label": { "en": [ "Example Book (pdf)" ] },
+      "format": "application/pdf"
+    }
+  ]
+}
+```
+
+## Classes
+
+The following sub-sections define the classes used in the IIIF Presentation Data Model. Only the semantics and core structural requirements are defined within this section, along with any deviations from other specifications that the classes might be drawn from. The descriptions do not define how the classes are used together, which is done in the Presentation API Overview.
+
+The name of each class is given at the top of its definition below. The exact string _MUST_ be used as the value of `type` in the JSON for the class.
+
+__Properties__<br>
+All resources _MUST_ have the following properties: [id](#id), and [type](#type).
+{: .note}
+
+### Collection
+{: #Collection}
+
+> `"type": "Collection"`
+
+A Collection is an ordered list of Manifests, and/or Collections.
+
+A Collection _MUST_ have an HTTP(S) URI given in `id`. It _MUST_ be able to be dereferenced to retrieve the JSON description.
+
+The members of a Collection are typically listed in the `items` property or in a series of Collection Pages. The members _MAY_ include both other Collections and Manifests, in order, to form a tree-structured hierarchy. Collections without any members are allowed but discouraged.  For example, a collection that had its last member removed might still be valuable to maintain as an empty collection.
+
+If there are too many members in the collection to fit within a single document, then the members _MAY_ be listed in Collection Pages. A reference to the first page of members is given in the `first` property, and the last page in the `last` property. In this case, the Collection _MUST NOT_ use the `items` property. Collections with pages _MUST_ have at least two pages, otherwise the members should be included in `items` on the Collection. Collection Pages _MUST NOT_ be embedded within the Collection for the same reason.
+
```

Comment:
> MUST have at least two pages, otherwise the members should be included

Is this to avoid having `first` and `last` pointing to the same resource?

I wonder if this is a SHOULD; for dynamically-generated collections, you could be flip-flopping between `items` having two pages and a list of items. E.g., 

https://example.com/iiif/manifests-added-today